Bottom: cb3bb45a40eeb62177b5e30d0f0a619962ad3931
Top:    e380b793890e52fd731b792fa255d2dd74fc4f21
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-23 22:19:18 +0000

Refresh of rename-resetfound-linesandpoly

---

diff --git a/src/find.c b/src/find.c
index 964e8ae..8c0f404 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2922,7 +2922,7 @@ LookupElementConnections (ElementType *Element, FILE * FP)
   /* reset all currently marked connections */
   User = true;
   TheFlag = FOUNDFLAG;
-  ResetConnections (true, FOUNDFLAG);
+  ClearFlagOnAllObjects (true, FOUNDFLAG);
   InitConnectionLookup ();
   PrintElementConnections (Element, FP, true);
   SetChangedFlag (true);
@@ -2943,7 +2943,7 @@ LookupConnectionsToAllElements (FILE * FP)
   /* reset all currently marked connections */
   User = false;
   TheFlag = FOUNDFLAG;
-  ResetConnections (false, FOUNDFLAG);
+  ClearFlagOnAllObjects (false, FOUNDFLAG);
   InitConnectionLookup ();
 
   ELEMENT_LOOP (PCB->Data);
@@ -2953,12 +2953,12 @@ LookupConnectionsToAllElements (FILE * FP)
       break;
     SEPARATE (FP);
     if (Settings.ResetAfterElement && n != 1)
-      ResetConnections (false, FOUNDFLAG);
+      ClearFlagOnAllObjects (false, FOUNDFLAG);
   }
   END_LOOP;
   if (Settings.RingBellWhenFinished)
     gui->beep ();
-  ResetConnections (false, FOUNDFLAG);
+  ClearFlagOnAllObjects (false, FOUNDFLAG);
   FreeConnectionLookupMemory ();
   Redraw ();
 }
@@ -3118,7 +3118,7 @@ LookupUnusedPins (FILE * FP)
   /* reset all currently marked connections */
   User = true;
   TheFlag = FOUNDFLAG;
-  ResetConnections (true, FOUNDFLAG);
+  ClearFlagOnAllObjects (true, FOUNDFLAG);
   InitConnectionLookup ();
 
   ELEMENT_LOOP (PCB->Data);
@@ -3143,7 +3143,7 @@ LookupUnusedPins (FILE * FP)
  * resets all used flags of pins and vias
  */
 bool
-ResetFoundPinsViasAndPads (bool AndDraw, int flag)
+ClearFlagOnPinsViasAndPads (bool AndDraw, int flag)
 {
   bool change = false;
 
@@ -3199,7 +3199,7 @@ ResetFoundPinsViasAndPads (bool AndDraw, int flag)
  * resets all used flags of LOs
  */
 bool
-ResetFoundLinesAndPolygons (bool AndDraw, int flag)
+ClearFlagOnLinesAndPolygons (bool AndDraw, int flag)
 {
   bool change = false;
 
@@ -3264,12 +3264,12 @@ ResetFoundLinesAndPolygons (bool AndDraw, int flag)
  * resets all found connections
  */
 bool
-ResetConnections (bool AndDraw, int flag)
+ClearFlagOnAllObjects (bool AndDraw, int flag)
 {
   bool change = false;
 
-  change = ResetFoundPinsViasAndPads  (AndDraw, flag) || change;
-  change = ResetFoundLinesAndPolygons (AndDraw, flag) || change;
+  change = ClearFlagOnPinsViasAndPads  (AndDraw, flag) || change;
+  change = ClearFlagOnLinesAndPolygons (AndDraw, flag) || change;
 
   return change;
 }
@@ -3339,7 +3339,7 @@ DRCFind (int What, void *ptr1, void *ptr2, void *ptr3)
           DumpList ();
           /* make the flag changes undoable */
           TheFlag = FOUNDFLAG | SELECTEDFLAG;
-          ResetConnections (false, TheFlag);
+          ClearFlagOnAllObjects (false, TheFlag);
           User = true;
           drc = false;
           Bloat = -PCB->Shrink;
@@ -3383,7 +3383,7 @@ DRCFind (int What, void *ptr1, void *ptr2, void *ptr3)
     }
   /* now check the bloated condition */
   drc = false;
-  ResetConnections (false, TheFlag);
+  ClearFlagOnAllObjects (false, TheFlag);
   TheFlag = FOUNDFLAG;
   ListStart (What, ptr1, ptr2, ptr3);
   Bloat = PCB->Bloat;
@@ -3393,7 +3393,7 @@ DRCFind (int What, void *ptr1, void *ptr2, void *ptr3)
       DumpList ();
       /* make the flag changes undoable */
       TheFlag = FOUNDFLAG | SELECTEDFLAG;
-      ResetConnections (false, TheFlag);
+      ClearFlagOnAllObjects (false, TheFlag);
       User = true;
       drc = false;
       Bloat = 0;
@@ -3444,7 +3444,7 @@ DRCFind (int What, void *ptr1, void *ptr2, void *ptr3)
   drc = false;
   DumpList ();
   TheFlag = FOUNDFLAG | SELECTEDFLAG;
-  ResetConnections (false, TheFlag);
+  ClearFlagOnAllObjects (false, TheFlag);
   return (false);
 }
 
@@ -3582,7 +3582,7 @@ DRCAll (void)
 
   TheFlag = FOUNDFLAG | DRCFLAG | SELECTEDFLAG;
 
-  if (ResetConnections (true, TheFlag))
+  if (ClearFlagOnAllObjects (true, TheFlag))
     {
       IncrementUndoSerialNumber ();
       Draw ();
@@ -3636,7 +3636,7 @@ DRCAll (void)
   END_LOOP;
 
   TheFlag = (IsBad) ? DRCFLAG : (FOUNDFLAG | DRCFLAG | SELECTEDFLAG);
-  ResetConnections (false, TheFlag);
+  ClearFlagOnAllObjects (false, TheFlag);
   TheFlag = SELECTEDFLAG;
   /* check minimum widths and polygon clearances */
   if (!IsBad)
diff --git a/src/find.h b/src/find.h
index 9644727..4cc68f8 100644
--- a/src/find.h
+++ b/src/find.h
@@ -52,9 +52,9 @@ void LookupElementConnections (ElementType *, FILE *);
 void LookupConnectionsToAllElements (FILE *);
 void LookupConnection (Coord, Coord, bool, Coord, int, bool AndRats);
 void LookupUnusedPins (FILE *);
-bool ResetFoundLinesAndPolygons (bool, int flag);
-bool ResetFoundPinsViasAndPads (bool, int flag);
-bool ResetConnections (bool, int flag);
+bool ClearFlagOnLinesAndPolygons (bool, int flag);
+bool ClearFlagOnPinsViasAndPads (bool, int flag);
+bool ClearFlagOnAllObjects (bool, int flag);
 void InitConnectionLookup (void);
 void FreeConnectionLookupMemory (void);
 void RatFindHook (int, void *, void *, void *, bool, int flag, bool);
